src/patternfly/components/DataToolbar/examples/DataToolbar.md

```diff
@@ -90,7 +90,6 @@ Several components in the following examples do not include functional and/or ac
 | -- | -- | -- |
 | `.pf-c-data-toolbar__item` | `<div>` | Initiates the toolbar component item. **Required** |
 | `.pf-c-data-toolbar__group` | `<div>` | Initiates the toolbar component group. |
-| `.pf-m-separator` | `.pf-c-data-toolbar__item` | Modifies item to a separator. |
```

Comment:
This is fine since it's experimental, but we could also save this change out and include it in https://github.com/patternfly/patternfly-next/issues/2621 since we're going to making this change globally, and it would keep the scope of this issue to just introducing the feature.